sdk/textanalytics/azure-ai-textanalytics/azure/ai/textanalytics/_response_handlers.py

```diff
@@ -231,6 +236,8 @@ def _get_task_type_from_error(error):
         return AnalyzeBatchActionsType.RECOGNIZE_ENTITIES
     if "entitylinking" in error.target.lower():
         return AnalyzeBatchActionsType.RECOGNIZE_LINKED_ENTITIES
+    if "sentimentanalysis" in error.target.lower():  # TODO is this correct?
```

Comment:
@iscai-msft sorry meant to ask you, I was unsure about this line - specifically if "sentimentanalysis" is correct. Do you know?

Reply:
discussed offline